tidy3d/plugins/smatrix/component_modelers/modal.py

```diff
@@ -217,6 +217,7 @@ def plot_sim(
         y: Optional[float] = None,
         z: Optional[float] = None,
         ax: Ax = None,
+        transpose: bool = False,
```

Comment:
Correct behavior of the `plot_sim()` function was verified visually (with `transpose=True`) using the  [SMatrix.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/SMatrix.ipynb) notebook example.